src/tyr/route_serializer_osrm.cc

```diff
@@ -535,6 +602,30 @@ std::string exits(const valhalla::DirectionsLeg_Maneuver_Sign& sign) {
   return exits;
 }
 
+valhalla::baldr::json::MapPtr serialize_incident(const TripLeg_Node_Incident& incident) {
+  auto metadata_json = json::map({});
+
+  {
+    uint64_t id = static_cast<uint32_t>(incident.id());
+    metadata_json->emplace("id", id);
```

Comment:
why not cast inline like the others below?

Reply:
I had a really long, opaque compiler error which didn't resolve until I did this. In hindsight, it seemed like the json document wouldn't allow both uint64 and uint32 inside.

With the increase of `id` to 64bits, this entire cast goes away.